SurfaceTopography/Generic/Curvature.py

```diff
@@ -111,7 +110,7 @@ def scale_dependent_curvature_from_area(topography, nbins=None):
     if len(nz) > 0:
         n = nz[0]
     # Important: The following expression relies on the fact that r is equally spaced!
-    return r[:n], np.sqrt(B[:n]) / r[:n] ** 2
+    return 2 * r[:n], np.sqrt(B[:n]) / r[:n] ** 2
```

Comment:
Please reflect this in the docstring: 

```python
    r"""
    Compute the two-dimensional, radially averaged scale-dependent curvature.
    The scale-dependent curvature is given by
       .. math::
         :nowrap:
         \begin{equation}
         h_\text{rms}^{\prime\prime}(\ell) = 4 \left[8A(\ell / 2) - 2A(\ell)\right]^{1/2}/\ell^2
         \end{equation}
    where :math:`A(\ell)` is the autocorrelation function.
    Parameters
    ----------
    topography : :obj:`SurfaceTopography` or :obj:`UniformLineScan`
        Container storing the uniform topography map
    nbins : int
        Number of bins for radial average. Bins are automatically determined
        if set to None. Note: Returned array can be smaller than this because
        bins without data points are discarded. (Default: None)
    Returns
    -------
    r : array
        Distances: length of the finite difference stencil (two pixels). (Units: length)
    curvature : array
        Curvature. (Units: 1/length)
    """  # noqa: E501

```